other/concat-exports.js

```diff
@@ -0,0 +1,88 @@
+const appendToFile = require('fs').appendFile
+const resolve = require('path').resolve
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+function capitalize(s) {
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
+}
+
+function dashToCamelCase(s) {
+  return s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+}
+
+const jsBuiltInObjectNames = [
```

Comment:
the `reserved` package wasn't updated in 3 years and is a bit outdated. I can use node.js to generate the list.